src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -101,6 +111,12 @@ class Kilosort4Sorter(BaseSorter):
         "scaleproc": "int16 scaling of whitened data, if None set to 200.",
         "save_preprocessed_copy": "save a pre-processed copy of the data (including drift correction) to temp_wh.dat in the results directory and format Phy output to use that copy of the data",
         "torch_device": "Select the torch device auto/cuda/cpu",
+        "bad_channels": "A list of channel indices (rows in the binary file) that should not be included in sorting. Listing channels here is equivalent to excluding them from the probe dictionary.",
+        "clear_cache": "If True, force pytorch to free up memory reserved for its cache in between memory-intensive operations. Note that setting `clear_cache=True` is NOT recommended unless you encounter GPU out-of-memory errors, since this can result in slower sorting.",
```

Comment:
Tiny question here. I really like this explanation. Do we want to make it clear for the user that this is being handled by KS4 and not by spikeinterface or do you think it doesn't matter?

Reply:
I think that then we should do it with every KS argument, no? :)

Reply:
Yeah I think so. The issue is as soon as we start mixing in spikeinterface commands and KS4 commands I can imagine users getting confused. So letting them know when it is a KS4 argument causing problems vs us will make the issue tracker easier to manage. But we could also say it is the user's responsibility to know what is KS4/spikeinterface. So although I lean toward being explicit one could argue that it is not necessary :)